playground/mocks/data/idp/idx/authenticator-verification-phone-options-voice-primary.json

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
this look same to `playground/mocks/data/idp/idx/authenticator-verification-phone-options.json`

Reply:
This one has voice as primary and sms as secondary in which case my labels are different for buttons, hence needed for e2e testing